src/plugins/intel_cpu/src/nodes/executors/dnnl/dnnl_utils.cpp

```diff
@@ -26,7 +26,8 @@ DnnlMemoryDescPtr makeTransposedWeightDescriptor(const DnnlMemoryDescPtr srcDesc
 MemoryPtr prepareWeightsMemory(const DnnlMemoryDescPtr srcWeightDesc,
                                const DnnlMemoryDescPtr dstWeightDesc,
                                const MemoryCPtr weightsMem,
-                               const ExecutorContext::CPtr context) {
+                               const ExecutorContext::CPtr context,
+                               const bool needShiftSignedToUnsigned) {
```

Comment:
change naming needShiftSignedToUnsigned to dynamicQuantization?

Reply:
I prefer to keep the naming, since dynamicQuantization is from the fullyconnect-caller's perspective, but the parameter name should reflect callee's behavior, other callers can require it for various other purposes.